gammapy/stats/variability.py

```diff
@@ -38,11 +46,12 @@ def compute_fvar(flux, flux_err):
        curves from active galaxies", Vaughan et al. (2003)
        https://ui.adsabs.harvard.edu/abs/2003MNRAS.345.1271V
     """
-    flux_mean = np.mean(flux)
+
+    flux_mean = np.nanmean(flux, axis=axis)
```

Comment:
I think there might be an issue here. 
If you use `np.nanmean` you will implicitly discard `nan` values but you are nevertheless using `len(flux)` as the number of points. You should first remove the `nan` values from the `flux` array (and do the same for the `flux_err` array.

Reply:
Would the correct thing be to treat `nan` as 0 instead? `np.nansum` (used in the function) does this.